data/hai/hai reveal 4 middle.txt

```diff
@@ -1141,28 +1141,43 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			`	Teeneep begins. "For the first time in three hundred years, we find ourselves beset on our borders by hostile forces. They have kidnapped five thousand of our people, and this council has been too paralyzed with fear of change to defend us. The Unfettered were once the defenders of all Hai, and I have convinced them to be so once again. Will you do what is right for our people, or will you be replaced?"`
+			`	At the word "replaced," murmurs rise and Sayari looks at you with deep concern. The moderator silences the room and asks you to testify. "Captain <last>, you have the unique vantage point of an accomplished outsider who has seen all parts of this conflict. What do you advise?`
+			`	You should carefully choose your words before speaking, as this may be a pivotal moment in history. This may spark a war - or prevent one.`
 			choice
-				`	(Get dinner.)`
-					goto dinner
-				`	(Decline and maybe eat with the diplomats.)`
-			`	There are some perfectly adequate meal places in the cafeteria area. There you manage to join Alondo and Remington, who have a surprisingly civilized conversation the entire time. Albeit mostly about work.`
-			`	You're still finishing up a little shortly after they leave, at which point a harried-looking Sayari appears and practically sprints across the room to you.`
-				goto alarm
-			label dinner
-			`	You join Teeneep and Sayari at a fusion restaurant about a block down the road from the 'Elders' Chambers' - which turns out to be the official name of the whole government complex.`
-			`	Having finished your main meal, and while waiting for some dessert that Teeneep has ordered, Sayari pulls out her datapad. "You know, while we're waiting. It's probably a good idea to check the damage. I don't know if your plan is actually going to have the desired effect, you know. I mean, seriously, Tee, what will you do if it doesn't? Will the Hai actually have to shed our brethren's blood with lethal intent?"`
-			`	Teeneep just sips her drink and shakes her head. "Oh, Sayari, you really are getting too long in the tooth for domestic politics these days."`
-			`	A moment later Sayari squeaks and looks up in horror. "You triggered an election?! This wasn't just making it public, you had the petition for redress ready to go all along!"`
-			`	She scrolls down on her pad several times and begins to mutter, "Oh dear, oh no." She lapses into Hai for a moment before returning. "It hit the trending issues board, and triggered the safety policy alert response. People got direct notifications that this was an issue. There are over two billion supporting votes for an injunctive action and it's only been two hours... this is what you hoped would happen, isn't it?"`
-			`Teeneep places a gentle hand on Sayari's arm. "Better this than war."`
-			label alarm
+				`	"You can trust the Unfettered to defend you, but you must be careful to avoid a war with the humans. That would be catastrophic for everyone."`
```

Comment:
I really like that they're asking your opinion for the UHai
Mb they could ask for the kidnapping threat as well? Since you're most expert in that, and actually saw victims first-hand

Idk why FL would be an UHai expert more than they are honestly, but kidnapping is where the expertise is at (though doesnt hurt to ask for UHai sincerity)

Reply:
I meant for the moderator's question to imply that he's asking about the full situation, including the kidnappings and the tensions with human governments. Perhaps I need to carefully reword his question.

Reply:
I just feel like thats where they should ask your opinion mostly

Reply:
Yes, that makes sense. I'll have them ask that before they ask about the Unfettered decision.

Reply:
I added a question to the player about the kidnappings.